src/Forms/EditFormFactory.php

```diff
@@ -103,4 +100,22 @@ protected function namespaceFields(FieldList $fields, array $context)
             $field->setName($namespacedName);
         }
     }
+
+    /**
+     * Remove the pseudo namespaces that were added in namespaceFields()
+     */
+    public function removeNamespaceFromFields(FieldList $fields, array $context): void
```

Comment:
There's no test for `namespaceFields()` nor for the old `removeNamespacesFromFields()` method on `ElementalAreaController`... these methods are both indirectly tested already.
But to avoid ping pong, I'll add tests for both of these methods after rebasing on top of the CMS 5 PR.

Reply:
Done